### DISCUSSION TITLE
滚动条宽高最大值

### DISCUSSION THREAD

**Discussion by minosss** [tusen-ai/naive-ui#3376] [General]
滚动条设置了

```css
top: 2px;
bottom: 2px;
```

最大值是不是 yRailSize 才对

https://github.com/TuSimple/naive-ui/blob/035611e9e9fed99cd6a89e43360c4c86e413e7b9/src/_internal/scrollbar/src/Scrollbar.tsx#L171-L174

**Comment by 07akioni** [comment]
感觉你说的好像有道理
